webserver/webserver/src/main/java/io/helidon/webserver/ServerConfiguration.java

```diff
@@ -549,6 +549,10 @@ public Builder config(Config config) {
         private SocketConfiguration.Builder configureSocket(Config config, SocketConfiguration.Builder soConfigBuilder) {
 
             config.get("port").asInt().ifPresent(soConfigBuilder::port);
+            // we use `host` in every MP example and it should be supported in SE as well
+            config.get("host").asString()
+                    .map(this::string2InetAddress)
+                    .ifPresent(soConfigBuilder::bindAddress);
             config.get("bind-address")
                     .asString()
                     .map(this::string2InetAddress)
```

Comment:
Maybe format `.asString()` to new line so its consistent with to bind-address fetching

Reply:
Fixed